.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- Note: Lines with this <!-- syntax are comments and will not be visible in
+     your pull request. You can safely ignore or remove them. -->
+
+### Pull Request Prelude
+
+<!-- Thank you for working on improving The Forgotten Server! -->
+<!-- Please complete these steps and check the following boxes by putting an `x`
+     inside the [brackets] before filing your Pull Request. -->
+
+- [ ] I have followed [proper The Forgotten Server code styling][code].
+- [ ] I have read and understood the [contribution guidelines][cont] before making this PR.
+- [ ] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.
+
+### Changes Proposed
+
+<!-- Describe the changes that this pull request makes. -->
+
+**Issues addressed:** <!-- Write here the issue number, if any. -->
+
+
+<!-- You can safely ignore the links below:  -->
+
+[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
+[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide
```

Comment:
Will these be visible? Do they need to be manually removed or is it some sort of GitHub magic I'm not aware of?

I don't think you've used the PR template here :D

Reply:
it will look like this:
![dsa](https://user-images.githubusercontent.com/4684880/105646083-98f3c400-5e7c-11eb-83dd-8685692c8411.png)
([example](https://github.com/HerculesWS/Hercules/pull/2918))
([source where I took the template](https://github.com/HerculesWS/Hercules/blob/stable/.github/PULL_REQUEST_TEMPLATE.md)) (click RAW)